docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -0,0 +1,155 @@
+      ```
+
+2. Navigate to the client directory. 
+
+      ```shell
+      cd miden-client
+      ```
+
+3. Modify the configuration file at `./miden-client.toml` to point to the remote Miden node. 
+
+      In the `[RPC]` section, replace the `endpoint = { host: }` field with the address provided by the Miden team.
+
+      ```toml
+      [rpc]
+      endpoint = { protocol = "http", host = "<NODE_IP_ADDRESS>", port = 57291 }
+
+      [store]
+      database_filepath = "store.sqlite3"
+      ```
+
+4. Build and install the client using cargo:
+
+      ```shell
+      cargo install --features testing,concurrent --path .
+      ```
+
+5. Check you can use the help flag.
+
+      ```shell
+      miden-client --help
+      ```
+
+## Create a new Miden account
+
+1. Create a new account called `basic-immutable` using the following command:
+
+      ```shell
+      miden-client account new basic-immutable
+      ```
+
+2. List all created accounts by running the following command:
+
+      ```shell
+      miden-client account -l
+      ```
+
+3. You should something like this:
+
+      ![Result of listing miden accounts](../../img/get-started/miden-account-list.png)
+
+Save the account ID for a future step.
+
+## Request tokens from the public faucet
+
+1. To request funds from the faucet navigate to the following website: [Miden faucet website](https://ethdenver.polygonmiden.io/).
+
+2. Copy the **Account ID** printed by the `miden-client account -l` command in the previous step. 
+
+3. Paste this id into the **Request test POL tokens** input field on the faucet website and click **Send me 100 tokens!**. 
+
+4. After a few seconds your browser should download - or prompt you to download - a file called `note.mno` (mno = Miden note). It contains the funds the faucet sent to your address.
+
+5. Save this file on your computer, you will need it for the next step. 
+
+## Import the note into the Miden client
+
+1. Import the note that you have received using the following commands: 
+
+      ```shell
+      miden-client input-notes -i <path-to-note>
+      ```
+
+2. You should see something like this:
+
+      ```sh
+      Succesfully imported note 0x0ff340133840d35e95e0dc2e62c88ed75ab2e383dc6673ce0341bd486fed8cb6
+      ```
+
+3. Now that the note has been successfully imported, you can view the note's information using the following command: 
+
+      ```shell
+      miden-client input-notes -l
+      ```
+
+4. You should see something like this:
+
+      ![Result of viewing miden notes](../../img/get-started/note-view.png)
+
+!!! tip "The importance of syncing"
+      - As you can see, the listed note is lacking a `commit-height`. 
+      - This is because you have received a note off-chain but have not yet synced your view of the rollup to check that the note is valid and exists at the rollup level. 
+      - Syncing is essential to prevent double-spends and make sure that you consume notes that have not yet been nullified. 
```

Comment:
You should not be able to consume the note if you don't sync. Just because I read this sentence as if the user would not sync a double spend can happen

Reply:
i think a **can** resolves it probably

Reply:
Actually, could you rewrite that line please as I'm not sure what you intend @Dominik1999

Reply:
Maybe, but I think this sentence is wrong

`Syncing is essential to prevent double-spends`

Double spends are prevented by the nullified check at the operator level after a tx has been done

Reply:
reworded a little but good, thank you